tests/plugins/tool/clang-format_tool_plugin/test_clang-format_tool_plugin.py

```diff
@@ -230,6 +230,8 @@ def test_clang_format_tool_plugin_scan_different_binary():
 def test_clang_format_tool_plugin_scan_custom_version():
     """Test that issues is empty when a custom version is specified."""
     cftp = setup_clang_format_tool_plugin()
```

Comment:
do we want to test for the existence of these here just like in the test_clang-tidy file?
```suggestion
    cftp = setup_clang_format_tool_plugin()
    if not cttp.command_exists("cmake"):
        pytest.skip("Can't find CMake, unable to test clang_tidy plugin")
    elif not cttp.command_exists("clang-format"):
        pytest.skip("Can't find clang-format, unable to test clang_format plugin")
```

Reply:
[57351ff](https://github.com/sscpac/statick/pull/444/commits/57351ffcbfe56be93d8794a338f1941484340704)

I would have taken the suggestion but it has some typos (`tidy` vs `format`).